helper-cli/src/main/kotlin/commands/CreateFlatAnalyzerResultCommand.kt

```diff
@@ -0,0 +1,181 @@
+import com.fasterxml.jackson.module.kotlin.readValue
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.helper.utils.writeOrtResult
+import org.ossreviewtoolkit.model.AnalyzerResult
+import org.ossreviewtoolkit.model.AnalyzerRun
+import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.Hash
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageLinkage
+import org.ossreviewtoolkit.model.PackageReference
+import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.Repository
+import org.ossreviewtoolkit.model.Scope
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.config.Excludes
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+import org.ossreviewtoolkit.model.config.ScopeExclude
+import org.ossreviewtoolkit.model.config.ScopeExcludeReason
+import org.ossreviewtoolkit.model.orEmpty
+import org.ossreviewtoolkit.utils.common.expandTilde
+
+internal class CreateFlatAnalyzerResultCommand : CliktCommand(
+    "A command which turns a simple definition file into an analyzer result."
+) {
+    private val simpleDefinitionFile by option(
+        "--simple-definition-file", "-i",
+        help = "The simple definition file to read the project definition from."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val ortFile by option(
+        "--ort-file", "-o",
+        help = "The ORT file to write the generated synthetic analyzer result to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val definitionFile = FileFormat.forFile(simpleDefinitionFile).mapper.readValue<SimpleDefinitionFile>(
+            simpleDefinitionFile
+        )
+
+        val projectVcs = VcsInfo(
+            type = definitionFile.vcsType ?: VcsType.UNKNOWN,
+            url = definitionFile.vcsUrl.orEmpty(),
+            revision = definitionFile.vcsRevision.orEmpty(),
+            path = definitionFile.vcsPath.orEmpty()
+        )
+
+        val projectName = definitionFile.name?.takeUnless { it.isBlank() } ?: DEFAULT_PROJECT_NAME
+
+        val project = Project.EMPTY.copy(
+            id = Identifier("Unmanaged::$projectName:"),
```

Comment:
Since the (only) contained project is hard-coded to "Unmanaged", maybe also a command name like `CreateUnmanagedProject` could make sense.

Reply:
Speaking of hard-coding, maybe also better use `${PackageManager.ALL["Unmanaged"].type}` instead of `Unmanaged` here, to document that this is a type of an existing package manager and not a new invention.